projects/uitdatabank/models/event-contactPoint-put.json

```diff
@@ -1,4 +1,53 @@
 {
-  "$ref": "./common-contactPoint.json",
-  "title": "event.contactPoint.put"
+  "title": "event.contactPoint.put",
+  "type": "object",
+  "properties": {
+    "phone": {
+      "type": "array",
+      "description": "List of phone numbers for contact purposes (can be empty)",
+      "minItems": 0,
+      "items": {
+        "type": "string",
+        "description": "Phone number for contact purposes.",
+        "minLength": 1,
+        "pattern": "\\S"
```

Comment:
Are these min-length and "at least one non-whitespace character required" checks also in the current code for this endpoint? (Or do you get a 500 if you pass an empty string or a string with only whitespace?) If not I'd leave them out.